pages/getting-started/sending-domains/sending-from-dreamhost.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Sending from a custom domain - DreamHost
+description: A step-by-step guide through setting up a custom domain to send your newsletter with DreamHost
+---
+
+import Layout from "../../../components/Layout";
+
+export const meta = {
+  title: "Sending from a custom domain - DreamHost",
+  description:
+    "A step-by-step guide through setting up a custom domain to send your newsletter with DreamHost",
+};
+export default ({ children }) => <Layout meta={meta}>{children}</Layout>;
+
+# Sending from a Custom Domain - DreamHost
+
```

Comment:
Can we add a notice that has a link to the [sending from a custom domain](https://docs.buttondown.email/getting-started/sending-from-a-custom-domain) doc to explain why folks would want to do this?